source/publishing_data/05_processing_data/processors/expression.rst

```diff
@@ -1,273 +1,352 @@
+- ``3``
+- ``2.5``
+- ``"Hello"`` or ``'Hello'``
+- ``'A bigger sentence'``
+- ``true`` or ``false``
+- ``pi``, ``PI`` or ``Pi``
+- ``e`` or ``E``
 
-.. list-table::
-   :header-rows: 1
+Operators
+~~~~~~~~~
 
-   * * Operator type
-     * Operators
-   * * Unary operators
-     * +, -, not, ! (not) , ! (factorial), ^ (power)
-   * * Binary operators
-     * +, -, ``*``, /, % (euclidean division), and, &&, or, ||, >, <, >=, <=, ==, != (evaluates to ``True`` or ``False``), &
-       (concatenation of strings, evaluates to a string)
-   * * Ternary operators
-     * op1 ? op2 : op3 (conditional statement). Please note that op3 is optional, so you can write op1 ? op2.
+Operators are symbols that behave generally like functions but are used with a more natural syntax.
 
-In the table below are listed the available functions:
+The Expression processor supports 3 kinds of operators, depending on the number of parameters around them:
+
+- **unary operators** can be used as prefixes or suffixes to alter the value of 1 expression,
+- **binary operators** can be arithmetic operators to perform a calculus between 2 expressions, or boolean operators to compare the result of 2 expressions,
+- the **ternary operator**, to convert a conditional expression to either 1 of 2 possible results.
+
+.. admonition:: Note
+   :class: note
+
+   Operator precedence works in the following order: factorial, exponential, sign, euclidian division, function evaluation, multiplication/division, addition/substraction, concatenation, not, comparison, and, or, ternary operator, ternary operator without else.
+
+Unary operators
+^^^^^^^^^^^^^^^
 
 .. list-table::
    :header-rows: 1
 
-   * * Function type
-     * Functions
-   * * Without operand
-     * now
-   * * Single operand
-     * abs, acos, acosh, asin, asinh, atan, atanh, capitalize, capitalize_all, ceil, cos, day, dayofweek, degrees, empty, exp, factorial, floor, fromtimestamp, gamma, hour, isalnum, isalpha, isdecimal, isdigit, islower, isnumeric, isupper, length, log, log10, lower, minute, month, normalize, quarter, quartertodaterange, radians, random, round, second, sigmoid, sin, sqrt, tan, tolowercase, touppercase, trunc, upper, week, year
-   * * Two operands
-     * add_days, add_hours, add_minutes, add_months, add_seconds, add_years, contains, day, dayofweek, distance, div, endswith, gcd, geopoint2d_towgs84, geoshape_towgs84, hour, match, max, microsecond, min, minute, mod, month, pow, quarter, quartertodaterange, random, round, second, startswith, substring, week, year
-   * * Three operands
-     * center, datediff, ljust, replace, rjust, substring
+   * * Operator
+     * Description
+     * Example
+   * * ``-``
+     * Prefix that negates the following value
+     * ``- 4``, ``- [expression]``
+   * * ``not``, ``!`` (not)
+     * Boolean operator that inverts the following condition
+     * * ``not true`` or  ``!true`` returns ``false``
```

Comment:
`not` doesn't work like an operator but as a method: `not('x')`

same usage as `!()` that is correct 2 lines below

Reply:
Good catch 👌